makefiles/in_chroot/install.sh

```
@@ -8,15 +8,17 @@ cd /opt/axiom-firmware
+# use fixed mirror, because the alarm loadbalancing and some mirrors are broken
+echo 'Server = http://de3.mirror.archlinuxarm.org/$arch/$repo' > /etc/pacman.d/mirrorlist
 pacman-key --init
 pacman-key --populate archlinuxarm
-pacman --noconfirm --needed -Syu
-pacman --noconfirm -R linux-zedboard || true
+pacman --noprogressbar --noconfirm --needed -Syu
+pacman --noprogressbar --noconfirm -R linux-zedboard || true
 
 # install dependencies
-pacman --noconfirm --needed -S $(grep -vE "^\s*#" makefiles/in_chroot/requirements_pacman.txt | tr "\n" " ")
+pacman --noprogressbar --noconfirm --needed -S $(grep -vE "^\s*#" makefiles/in_chroot/requirements_pacman.txt | tr "\n" " ")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)